### ISSUE TITLE
Request for Additional Materials on MFRC-NET

### ISSUE THREAD

**Issue by pandrikopoul** [WeizhuoYang/MFRC-Net#1]
Dear @WeizhuoYang  ,

My name is Panagiotis Andrikopoulos, and I am a master's student in Data Science at Utrecht University. As part of my thesis project on deep learning for Motor Imagery classification, I have read your paper:

Li, X., Yang, Z., Tu, X., Wang, J., & Huang, J. (2024). MFRC-NET: Multi-Scale Feature Residual Convolutional Neural Network for Motor Imagery Decoding. IEEE Journal of Biomedical and Health Informatics, 1–11. https://doi.org/10.1109/jbhi.2024.3467090

I have also reviewed the provided code from the corresponding GitHub repository. However, since the repository mainly focuses on the model architecture, I was wondering if you could kindly share additional parts of the code, particularly those related to:

-Filtering the raw EEG data to produce the multi-view structure
-The specific loss function used in your implementation

Also, I want to ask you about your exact data split for the Cross-session evaluation. Did you use the 1st session for training and the remaining 4 for testing? Did you train a separate model for each subject  (is it subject-dependent Cross-session evaluation)? 

This would greatly help me in replicating your results on the SHU dataset. I would truly appreciate any assistance you can provide.
Thank you for your time and consideration. 
I look forward to your response.
Best regards,
Panagiotis Andrikopoulos

**Comment by pandrikopoul** [comment]
@WeizhuoYang,
Kind Reminder!

**Comment by pandrikopoul** [comment]
@WeizhuoYang

**Comment by WeizhuoYang** [comment]
I am very glad to have received your message. I will organize and upload the code as soon as possible. You can choose any test session from Session 2 to Session 5, with Session 1 being the training session.

**Comment by pandrikopoul** [comment]
Great, I am looking forward to run your code! @WeizhuoYang 

The 63.9% cross-session accuracy you achieved for the SHU dataset, is the average accuracy for all the 25 subjects? Meaning that you trained a different model for each subject on its first session and testing on the remaining 4 sessions of the same subject, and then you get 63.9% average accuracy for all the 25 subjects ?

**Comment by pandrikopoul** [comment]
@WeizhuoYang, kind reminder!

**Comment by pandrikopoul** [comment]
Dear @WeizhuoYang,

Just a kind reminder that I’m waiting for the code and answers to my last questions so I can fully reproduce your results. Given my time constraints, your prompt help would be greatly appreciated.

Thank you,
Panagiotis Andrikopoulos
